File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Unntak.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import lombok.Data;

@JsonInclude(Include.NON_NULL)
@Data
public class Unntak {

    private String startdatoansattforsikret;

    private Grunnlag grunnlag;

    private SpesielleOmstendigheter spesielleomstendigheter;

    private String startdatokontraktansettelse;

    private String begrunnelse;

    private String a1grunnlag;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Unntak.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Unntak(
    var startdatoansattforsikret: String? = null,
    var grunnlag: Grunnlag? = null,
    var spesielleomstendigheter: SpesielleOmstendigheter? = null,
    var startdatokontraktansettelse: String? = null,
    var begrunnelse: String? = null,
    var a1grunnlag: String? = null
)
